sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -136,11 +136,11 @@ describe("Listing methods - PagedAsyncIterableIterator", function(): void {
 
   [
     "getQueues",
-    "getQueuesRuntimeInfo",
+    "getQueuesRuntimeProperties",
     "getTopics",
-    "getTopicsRuntimeInfo",
+    "getTopicsRuntimeProperties",
     "getSubscriptions",
-    "getSubscriptionsRuntimeInfo",
+    "getSubscriptionsRuntimeProperties",
     "getRules"
   ].forEach((methodName) => {
     describe.only(`${methodName}`, () => {
```

Comment:
Oops! I think I missed removing it.
```suggestion
    describe(`${methodName}`, () => {
```